opennebula/defaults.yaml

```diff
@@ -99,10 +74,69 @@ kernel:
       controller_sshpubkeys: {}
 os_family:
   Debian:
+    oneadmin:
+      optional_groups:
+        - kvm
+        - libvirt
+        - ssl-cert
+    controller:
+      pkgs:
+        - opennebula
+        - opennebula-tools
+        - mysql-client
     compute_node:
       pkgs:
         - opennebula-node
+    oneflow:
+      pkgs:
+        - opennebula-flow
+        - ruby-treetop
+        - ruby-polyglot
+    onegate:
+      pkgs:
+        - opennebula-gate
+    sunstone:
+      pkgs:
+        - opennebula-sunstone
+        - python-numpy
   RedHat:
+    controller:
+      pkgs:
+        - opennebula-server
+        - mariadb
+        - rubygem-nokogiri
+        - rubygem-json
+        - rubygem-rack
+        - rubygem-sinatra
+        - rubygem-thin
+        - rubygem-zendesk_api
+        - rubygem-sqlite3
+        - rubygem-sequel
+        - rubygem-amazon-ec2
+        - rubygem-uuidtools
+        - rubygem-curb
+        - rubygem-net-ldap
+        - rubygem-builder
+        - rubygem-trollop
+        - rubygem-treetop
+        - rubygem-parse-cron
+        - rubygem-aws-sdk
+        - rubygem-ox
+        - rubygem-mysql
+        - rubygem-softlayer_api
+        - rubygem-configparser
+        - rubygem-azure
```

Comment:
Do you mean, that somebody could write the rubygem-list into the merged pillar data, if he build the packages like me?
Because this packages are not publicly available and without the packages the formula would fail.